x-pack/metricbeat/module/googlecloud/billing/_meta/docs.asciidoc

```diff
@@ -0,0 +1,24 @@
+`billing` metricset is designed for collecting billing metrics from Google Cloud
+BigQuery daily cost detail table. BigQuery is a fully-managed, serverless data
+warehouse.
+Cloud Billing export to BigQuery enables you to export detailed Google Cloud
+billing data (such as usage, cost estimates, and pricing data) automatically
+throughout the day to a BigQuery dataset that you specify. Then you can access
+your Cloud Billing data from BigQuery for detailed analysis using Metricbeat.
+Please see https://cloud.google.com/billing/docs/how-to/export-data-bigquery[export
+cloud billing data to BigQuery] for more details on how to export billing data.
+
+[float]
+=== Configuration example
+[source,yaml]
+----
+- module: googlecloud
+  metricsets:
+    - billing
+  period: 24h
+  project_id: "your project id"
+  credentials_file_path: "your JSON credentials file path"
+  dataset_id: "dataset id"
+  table_pattern: "table pattern"
+  cost_type: "regular"
```

Comment:
I miss a few explanations on what these parameters are. I'm personally not very familiar with this service so I don't fully understand what to put here

Reply:
Done. Thank you!